### ISSUE TITLE
GMail in other language than English

### ISSUE THREAD

**Issue by msiemens** [leggett/simplify#3]
When using another language than English for the GMail interface, the search button doesn't work. For example, in the German interface the `aria-label` of the search button is `In Nachrichten suchen`. This issue also applies to a few of the other CSS rules.

**Comment by leggett** [comment]
Correct. The extension is marked as english only for now for this reason. I should have been more up front about this limitation in my announcement. The aria-labels are stable where as the classnames change frequently. I've got on my list to find a way around this so it works in all languages.

**Comment by leggett** [comment]
Just for the search box, other users have reported that you can still access search via the "/" keyboard shortcut when your interface language is something other than English.

**Comment by msiemens** [comment]
All right, just wanted to report this bug in case it wasn't known yet 🙃

**Comment by leggett** [comment]
Just submitted an update with a fix to the Chrome web store. Says it will take up to 60 minutes to process and then however long it takes to auto-update in your browser. 

I suspect this will be a temporary fix as I'm using the classname instead of the aria-label and the classname on that element tends to change every week or so. I'll work on a more permanent fix this weekend.

**Comment by leggett** [comment]
Annnd then I realized I didn't totally fix it. I think I've got it now. Re-open the issue if it isn't fixed in v1.0.3 (publishing now).

**Comment by msiemens** [comment]
It's working now, thanks! 🙃
